### PR TITLE
SWI-2832 Migrate to OpenAPI Generator `v7.0.0`

### PR DESCRIPTION
**Pull request by ajrice6713**

### CODE REVIEW

File: test/integration/test_calls.py [hunk 1]
```diff
@@ -118,62 +119,65 @@ def test_create_call(self):
...
         """Validate a forbidden (403) request
         """
         with self.assertRaises(ForbiddenException) as context:
-            self.forbidden_api_instance.create_call(BW_ACCOUNT_ID, self.testCallBody, _return_http_data_only=False)
+            self.forbidden_api_instance.create_call(BW_ACCOUNT_ID, self.testCallBody)
 
         self.assertApiException(context, ForbiddenException, 403)
 
 
     def test_get_call_state(self):
```
Comment (ckoegel) [inline test/integration/test_calls.py:166]: do we need to add logic here to deal with the high latency for this operation?

Comment (ajrice6713) [reply test/integration/test_calls.py:166]: all the tests are passing as is so i say leave it for now 🤷

Comment (ajrice6713) [reply test/integration/test_calls.py:166]: We want to migrate to mocking anyways right?

**Comment by bw-github-sdlc-enforcer** [comment]
@noahg1 the `github-sdlc-enforcer` check was triggered.

File: test/integration/test_calls.py (hunk 1 above)
Comment (matthewkmartin) [reply test/integration/test_calls.py:166]: sad manteca

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-2022, windows-2019, ubuntu-18.04, ubuntu-20.04]
-        python-version: [3.8, 3.9, "3.10"]
+        python-version: [3.8, 3.9, "3.10", "3.11"]
```
Comment (hamermike) [inline .github/workflows/test.yaml:25]:
```suggestion
        python-version: [3.8, 3.9, 3.10, 3.11]
```

**Review by hamermike — CHANGES_REQUESTED** [review]

File: test/test_account_statistics.py
Comment (juliajanu) [inline test/test_account_statistics.py:1]: Why are all of these new unit tests added? `test_account_statistics.py` exists here and in `test/unit`.

File: .github/workflows/test.yaml (hunk 2 above)
Comment (ajrice6713) [reply .github/workflows/test.yaml:25]: That causes the runners to explode

Comment (hamermike) [reply .github/workflows/test.yaml:25]: That sounds like a valid reason to not accept my change.

**Review by hamermike — DISMISSED** [review]

**Review by hamermike — APPROVED** [review]

**Review by juliajanu — APPROVED** [review]

**Review by matthewkmartin — APPROVED** [review]

**Pull request merged**